TrueCraft.API/Configuration.cs

```diff
@@ -3,8 +3,17 @@
 
 namespace TrueCraft.API
 {
+    /// <summary>
+    /// Abstract base class for configurations read from YAML files.
+    /// </summary>
     public abstract class Configuration
     {
+        /// <summary>
+        /// Creates and returns a new configuration read from a YAML file.
+        /// </summary>
+        /// <typeparam name="T">The configuration type.</typeparam>
+        /// <param name="configFileName">The path to the YAML file.</param>
+        /// <returns></returns>
```

Comment:
Oh, nevermind, I guess this is YAML. Why does the API layer enforce YAML? Weird.